webapp/src/store/Store.ts

```diff
@@ -18,4 +70,13 @@ export class Store {
     }
     return projectStore;
   }
+
+  public async loadFromDisk(): Promise<void> {
+    try {
+      const json = await fs.readFile(FILE_PATH);
+      this.initialState = JSON.parse(json.toString());
+    } catch (err) {
+      // Do nothing. It's fine to start from scratch sometimes.
+    }
```

Comment:
Hiding errors is going to make bugs hard to discover and diagnose. We should probably log this error so that instance maintainers can find it. We can defer that to a separate issue.